src/BoundingBox.cpp

```diff
@@ -47,7 +47,9 @@ glm::vec3 BoundingBox::calcReflNormal(BoundingBox& other) {
    }
    if((this->min_.y - epsilon <= other.max_.y && this->min_.y + epsilon >= other.max_.y) ||
            (this->max_.y - epsilon <= other.min_.y && this->max_.y + epsilon >= other.min_.y)){
-       normal = glm::vec3(0.0, 1.0, 0.0);
+
+       // TODO(rgarmsen2295): Hack, should be y as normal, figure out why collisions are matching x instead of y only
+       normal = glm::vec3(1.0, 0.0, 0.0);

```

Comment:
Nah, I appreciate it. The problem doesn't make sense, so it's likely to be a non-sensical root issue like that.  Keep spittin' ballin' ideas like you Ballmer on the Ballmer peak, broski xD